docs/pages/link.js

```diff
@@ -68,9 +305,134 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+      checked
+      id="3"
+      onChange={() => {}}
+      size="sm"
+    />
+    <Label htmlFor="3">
+      <Text>
+        Use your activity to improve ads you see about Pinterest on other sites or apps you may visit.{' '}
+        <Link accessibilityLabel="Learn more about third-party analytics" inline href="https://www.pinterest.com/_/_/help/article/third-party-analytics-or-advertising-providers-pinterest-uses-or-allows">
+          Learn more
+        </Link>
+      </Text>
+    </Label>
+  </Flex>
+
+</Flex>
+`}
+          />
+          <MainSection.Card
+            defaultCode={`
+            <Table accessibilityLabel="See what's possible with access tiers" maxHeight={200}>
+  <Table.Header sticky>
+    <Table.Row>
+      <Table.HeaderCell />
+      <Table.HeaderCell>
+        <Text weight="bold">Trial</Text>
+      </Table.HeaderCell>
+      <Table.HeaderCell>
+        <Text weight="bold">Standard</Text>
+      </Table.HeaderCell>
+    </Table.Row>
+  </Table.Header>
+  <Table.Body>
+    <Table.Row>
+      <Table.Cell>
+        <Text weight="bold">Basic analytics</Text>
+      </Table.Cell>
+      <Table.Cell>
+        <Text>Included</Text>
+      </Table.Cell>
+      <Table.Cell>
+        <Text>Included</Text>
+      </Table.Cell>
+    </Table.Row>
+    <Table.Row>
+      <Table.Cell>
+        <Text weight="bold">Rate limits</Text>
+      </Table.Cell>
+      <Table.Cell>
+        <Flex direction="column" gap={2}>
+          <Text size="sm">1000/day</Text>
+          <Text>
+            <Link
+              accessibilityLabel="See rate limit details for trial package"
+              underline="always"
+              href="https://developers.pinterest.com/docs/api/v5/#tag/Rate-limits"
+            >
+              See details
+            </Link>
+          </Text>
+        </Flex>
+      </Table.Cell>
+      <Table.Cell>
+        <Flex direction="column" gap={1}>
+          <Text size="sm">Variable</Text>
+          <Text>
+            <Link
+              accessibilityLabel="See rate limit details for standard package"
+              underline="always"
+              href="https://developers.pinterest.com/docs/api/v5/#tag/Rate-limits"
+            >
+              See details
+            </Link>
+          </Text>
+        </Flex>
+      </Table.Cell>
+    </Table.Row>
+  </Table.Body>
+</Table>;
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Keyboard navigation"
+          description="Give Link keyboard focus with the tab key (or shift + tab when tabbing backwards). Activate Link with the enter/return key."
+        />
         <MainSection.Subsection
           title="Accessible Tab Link"
-          description={`Use \`accessibilitySelected\` and \`role="Tab"\` when using Link as a tab.`}
+          description={`Use \`accessibilitySelected\` and \`role="Tab"\` when using Link as a tab. However, don't use Link to create custom tabs, use [Tabs](/tabs) instead.`}
```

Comment:
nit: `role="tab"` (lowercase)

Though really this example needs to be improved, as we're both showing a use case that should use a different component, and a use case that doesn't actually match what the "tab" role is supposed to be used for.

We can consider an update out of scope for this PR, but please put it on your list of Link improvements to come up with an example of [an actual "tab" role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role) for this one.

Reply:
DONE typo